### PR TITLE
Bump revision to 8.11.0, add changelog

### PR DESCRIPTION
**Pull request by jkeiser**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,10 +1,25 @@
...
-* [pr#722](https://github.com/chef/ohai/pull/722) Detect paravirt amazon instances without hint files
-* [pr#714](https://github.com/chef/ohai/pull/714) Detect Azure using the Azure agent and DHCP options
-* [pr#724](https://github.com/chef/ohai/pull/725) Add support for the 2 latest EC2 metadata versions
+## [8.11.0](https://github.com/chef/ohai/tree/8.11.0) (2016-03-02)
+[Full Changelog](https://github.com/chef/ohai/compare/v8.10.0...8.11.0)
+
+**Implemented enhancements:**
+
+- Detect Azure on non-bootstrapped hosts [\#657](https://github.com/chef/ohai/issues/657)
+- chef-client hangs \(ohai discovery hangs\) when a server has a mounted nfs filesystem while the remote host does not respond [\#537](https://github.com/chef/ohai/issues/537)
```
Comment (tas50) [inline CHANGELOG.md]: I added the label to exclude this one so you'll want to regenerate the changelog now for this PR.

**Comment by jkeiser** [comment]
that was not merged :)

@tas50 changelog regenerated!

**Comment by tas50** [comment]
:+1:

**Comment by tas50** [comment]
@jkeiser Looks like this same change was already merged.  Can this get closed out?

**Pull request closed without merging**